serverless/eventing/channels/connecting-channels-sinks.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="connecting-channels-sinks"]
+= Connecting channels to sinks
+:context: connecting-channels-sinks
+toc::[]
+
+Events that have been sent to a channel from an event source or producer can be forwarded to one or more sinks by using _subscriptions_.
+You can create subscriptions by configuring a `Subscription` object, which specifies the channel and the sink (also known as a _subscriber_) that consumes the events sent to that channel.
```

Comment:
```suggestion
You can create subscriptions by configuring a `Subscription` object, which specifies the channel, and the sink (also known as a _subscriber_) that consumes the events sent to that channel.
```

Reply:
Just found it hard to parse when reading

Reply:
I think it sort of reads weirdly with the second comma 😅 will leave as is if that's OK